canonical/canonical.com
# Update /careers/people copy

## Done

- Update copy as per [doc](https://docs.google.com/document/d/1sj-WToRUa9N2c5brs8iAoOfOPHJCeHwvHByq8qcSgK4/edit)


## QA

- View the site locally in your web browser at: https://canonical-com-1001.demos.haus/careers/people
- See that the change was made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5550
